.cirrus.yml

```diff
@@ -0,0 +1,79 @@
+# Reference:
+#   - https://cirrus-ci.org/guide/writing-tasks/
+#   - https://cirrus-ci.org/guide/linux/
+#   - https://hub.docker.com/_/gcc/
+#   - https://hub.docker.com/_/python/
+
+#
+# Global defaults.
+#
+container:
+  image: python:latest
```

Comment:
@trexfeathers We could default to `gcc:latest` instead and then drop lines 43-44 in `linux_task`

Reply:
e71ae3607e57ec75888fa55a9abfacce8bc9f880